Lib/uuid.py

```diff
@@ -673,12 +680,26 @@ def _random_getnode():
     return random.getrandbits(48) | (1 << 40)
 
 
-_node = None
-
-_NODE_GETTERS_WIN32 = [_windll_getnode, _netbios_getnode, _ipconfig_getnode]
+if _LINUX:
+    _GETTERS = [_ifconfig_getnode, _ip_getnode,
+                          _arp_getnode, _lanscan_getnode]
+elif _DARWIN:
+    _GETTERS = [_arp_getnode, _ifconfig_getnode, _netstat_getnode]
+elif _WINDOWS:
+    _GETTERS = [_windll_getnode, _netbios_getnode, _ipconfig_getnode]
+elif _AIX:
+    _GETTERS = [_netstat_getnode]
+else:
+    _GETTERS = [_ifconfig_getnode, _ip_getnode,
+                          _arp_getnode, _lanscan_getnode, _netstat_getnode]
+if os.name == 'posix':
+    _NODE_GETTERS = [_unix_getnode] + _GETTERS
+elif os.name == 'nt':
+    _NODE_GETTERS = [_windll_getnode] + _GETTERS
```

Comment:
Is this `nt` branch needed given the `_WINDOWS` branch above?

Reply:
no, you spotted my error - thx.  (I forgot to remove _windll_getnode above.)

As one of my goals is to shorten the _GETTERS assignment line, quick question:

what reads better - having an additional statement only for 'posix', or also having an 'elif' for Windows?

Reply:
I decided - _GETTERS is without the "platform" special and will be added when _NODE_GETTERS is assigned.